### ISSUE TITLE
Improve layout and typography of the Getting-Started watermark

### ISSUE THREAD

**Issue by kieferrm** [Microsoft/vscode#12899]
- VSCode Version: 1.6 insiders
- OS Version: MacOS

See the following screenshot:
![screen shot 2016-09-28 at 08 56 25](https://cloud.githubusercontent.com/assets/4674940/18921450/aba20a7c-8559-11e6-9967-6ec3011c3cbb.png)

1. The blue boxes should be vertically centerline aligned.
2. The bold font is too dense and hard to read particularly as the contrast between the foreground and the background color is muted and thus the text looks blurred. Setting the font-weight to normal and using `letter-spacing` of around `0.04em` improves readability.
3. The keyboard shortcuts on the right are then still hard to parse. Putting them in a  `span` and using a `letter-spacing` of `0.15em` (or `0.1em`) and a left and right padding of `0.5em` improves this.

This is what it looks like with  the changes proposed in 2. and 3.
![screen shot 2016-09-28 at 9 46 35 am](https://cloud.githubusercontent.com/assets/4674940/18923281/899a8d26-8560-11e6-99be-28c80c581318.png)

**Comment by chrmarti** [comment]
Tracking horizontal alignment as #12779. The suggested changes improve readability for me. @bgashler1 What do you think?

**Comment by bgashler1** [comment]
@kieferrm I'd prefer to use less-heavy font-weight that is semi-bold so that it is still differentiated as the master of the detail on the right.  This would be `font-weight: 600`.  What do you think?

<img width="354" alt="screen shot 2016-09-28 at 10 37 01 am" src="https://cloud.githubusercontent.com/assets/11839736/18925131/86924658-8567-11e6-994f-de0f22e596d6.png">

**Comment by kieferrm** [comment]
@bgashler1 I personally prefer `normal` due to the blurring effect described above. However, I'm only one voice...

**Comment by bgashler1** [comment]
@kieferrm I get where you're coming from.  I realize now why this was blurring the font.  Chrome does synthetic bolding of fonts when there is not an explicit bold-designed font being used.  http://stackoverflow.com/questions/21332475/bold-fonts-in-google-chrome-are-blurry-font-face
